extensions/ffi/lpffiwrappers.pas

```diff
@@ -833,7 +838,7 @@ function LapeFFIStackParam(const Param: TLapeParameter; ABI: TFFIABI): Boolean;
     Result := (Param.VarType <> nil) and (Param.VarType.BaseType = ltDynArray)
 
     {$IF (FPC_VERSION >= 3) AND DECLARED(FFI_PASCAL)}
-      or ((ABI = FFI_PASCAL) and (Param.VarType.Size < SizeOf(Pointer)) and (Param.VarType.Size <> SizeOf(UInt16)))
+      or ((ABI = FFI_PASCAL) and (Param.VarType.Size < SizeOf(Pointer)));
```

Comment:
Semicolon not needed here, there's one below. Also not sure why the UInt16 size check was here before, but I'll trust  the tests.